src/renderer.c

```diff
@@ -215,54 +208,41 @@ static void font_clear_glyph_cache(RenFont* font) {
   }
 }
 
+// based on https://github.com/libsdl-org/SDL_ttf/blob/2a094959055fba09f7deed6e1ffeb986188982ae/SDL_ttf.c#L1735
+static unsigned long font_file_read(FT_Stream stream, unsigned long offset, unsigned char *buffer, unsigned long count) {
+  Sint64 amount;
```

Comment:
I'd prefer using standard types to SDL's, but even their documentation is mixing them...

Reply:
If you're good with it, we'll use uint64_t and the like.

Reply:
Yeah I'd be fine with `int64_t`. Still, just a nit.